doc/api/cli.md

```diff
@@ -154,53 +208,80 @@ Note: v8 options allow words to be separated by both dashes (`-`) or underscores
 For example, `--stack-trace-limit` is equivalent to `--stack_trace_limit`.
 
 ### `--tls-cipher-list=list`
+<!-- YAML
+added: v4.0.0
```

Comment:
I think that means that 4.0.0 is the right version to go with in the YAML comment. Does anyone disagree? /cc @addaleax @bengl @sam-github